util/Hexagon/README.md

```diff
@@ -0,0 +1,57 @@
+# Hexagon/Halide Instrumentation
+
+Adds timing and reporting to Hexagon code generated by a Halide Generator.
+
+## Input Files
+`instrumented_generator.h`
+`profiling.cpp`
+`logcat2json.cpp`
+
+## Building
+For now, build `profiling.cpp` with your DSP code. Later we will likely include this precompiled binary.
```

Comment:
We should also get this built and packaged as a library. Or, if it's small, we can inject it via `target_sources(INTERFACE)`, as we do with GenGen (`Halide::Generator`).